AirportsLatLongConverter.py

```diff
@@ -0,0 +1,20 @@
+import airportsdata

```

Comment:
where does this module come from?  does it have to be installed?  is it built-in to all Python installations?

Reply:
It's an installation that Gabe found. I should include it in the read me

Reply:
added the instructions to install the airportdata to the Readme